frontend/integration-tests/protractor.conf.ts

```diff
@@ -212,10 +212,12 @@ export const config = {
 };
 
 export const checkLogs = async () =>
-  (await browser
-    .manage()
-    .logs()
-    .get('browser')).map((log) => {
+  (
```

Comment:
Don't know that this extra space after the parens is necessary.  

I'd probably like to see this land at something like:



```javascript 
export const checkLogs = async () => (
  await browser
    .manage()
    .logs()
    .get('browser')
).map(log => {
  browserLogs.push(log);
  return log;
})
```

Reply:
Hmm, we might just need to research what kind of prettier rules we can adjust for this.